.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: python
+sudo: false
+
+addons:
+  apt:
+    packages:
+      - rpm
```

Comment:
Is that required?

Reply:
Ah right, that's for building the rpm package. All good!

Reply:
yep, that for rpm package, but this PR does not finish the packaging part though